core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -345,6 +364,14 @@ public int getValue() {
    */
   boolean isFile(String path) throws IOException;
 
+  /**
+   * Checks if the ufs path is one of the underlying physical UFSs.
+   *
+   * @param ufsPath the ufs path w/ only scheme and authority
+   * @return true if physical UFS is covered by this UFS, false otherwise
+   */
+  boolean isPathCovered(String ufsPath);
```

Comment:
Instead of this API, maybe we only need to be able to get the path, and then convert it to a physical path, to do comparisons with other physical paths?

Reply:
similar to calvin's comment above, i'm on board with replacing isPathCovered with getPhysicalPaths